src/main/java/uk/gov/hmcts/ccd/ApplicationParams.java

```diff
@@ -67,6 +67,9 @@
     @Value("${definition.cache.ttl.secs}")
     private Integer definitionCacheTTLSecs;
 
+    @Value("${daily.cache.ttl.secs}")
```

Comment:
would change this to `user.cache.ttl.secs`. Much more descriptive. Same for the property name

Reply:
done